### ISSUE TITLE
Bug(`sort-imports`): inline comment misplace after fix

### ISSUE THREAD

**Issue by zzxming** [azat-io/eslint-plugin-perfectionist#319]
### Describe the bug

If there are comments after import, sort-import will cause the comments to be misplaced

### Code example

configuration
```
rules: {
  'perfectionist/sort-imports': ['error', {
    order: 'asc',
    // all three types are have same bug
    type: 'natural',
  }],
},
```

code
```
import A from './a' // comment A
import C from './c' // comment C
import B from './b' // comment B
```

after fix
```
import A from './a' // comment A
import B from './b' // comment C
import C from './c' // comment B
```

Expected 
```
import A from './a' // comment A
import B from './b' // comment B
import C from './c' // comment C
```

### ESLint version

9.9.0

### ESLint Plugin Perfectionist version

3.8.0

### Additional comments

I read code and found that the `sort-imports` fix only modified the position of the import statement. This may be an expected behavior to prevent multiple import statements from being on the same line?

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by hugop95** [comment]
It seems that the recent PR https://github.com/azat-io/eslint-plugin-perfectionist/pull/312 has fixed this issue.

**Comment by azat-io** [comment]
Yes, I'll create release today at evening

**Comment by azat-io** [comment]
Fixed in 398ac13.
Releases in [v3.9.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.9.0).
